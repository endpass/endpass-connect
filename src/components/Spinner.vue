<template>
  <div class="spinner"/>
</template>

<script>
export default {
  name: 'Spinner',
};
</script>

<style lang="postcss">
@keyframes rotate {
  to {
    transform: rotate(360deg);
  }
}

.spinner {
  width: 32px;
  height: 32px;
  border-radius: 50%;
  border: 3px solid #6d1f966e;
  border-top: 3px solid #6d1f96;
  animation: rotate 750ms linear infinite;
}
</style>
