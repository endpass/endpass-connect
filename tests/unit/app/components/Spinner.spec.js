import { shallowMount } from '@vue/test-utils';
import Spinner from '@@/app/src/components/Spinner.vue';

describe('Spinner', () => {
  describe('render', () => {
    let wrapper;

    beforeEach(() => {
      wrapper = shallowMount(Spinner);
    });

    it('should correctly render Spinner component', () => {
      expect(wrapper.name()).toBe('Spinner');
      expect(wrapper.html()).toMatchSnapshot();
    });
  });
});
