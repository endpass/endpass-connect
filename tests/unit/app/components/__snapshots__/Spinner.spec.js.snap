// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner render should correctly render Spinner component 1`] = `<div class="spinner"></div>`;
